### DISCUSSION TITLE
Broken pipe (os error 32)" when running forge build --zksync on Foundry zkSync Edition

### DISCUSSION THREAD

**Discussion by Fardeen26** [Cyfrin/foundry-full-course-cu#2059] [Q&A] [answered]
Hi everyone,

I've recently installed the Foundry zkSync edition following the official documentation. Everything seemed to be working fine until I tried to compile my smart contract using the command:

```forge build --zksync```

I'm encountering the following error in the terminal:

```Broken pipe (os error 32)```

### Environment:

- Operating System: Windows (using the WSL 2)
- Foundry Version: (forge 0.2.0)
- zkSync Plugin Version: (forge 0.0.2)

**Comment by Shiva-Sai-ssb** [comment, marked as answer]
Hey @Fardeen26, refer to this issue (https://github.com/matter-labs/foundry-zksync/issues/453) on the foundry-zksync repo.

As suggested in the issue, try to use the fixed version of solidity in the pragma declaration. Let's see if this helps you solve the issue.

**Reply by Fardeen26** [reply]
Thank you, Shiva! Referring to the issue and using a fixed version of Solidity in the pragma declaration solved the problem. I appreciate your help ♥

**Reply by EngrPips** [reply]
Awesome

**Reply by 0xdhiman** [reply]
it works

**Reply by EngrPips** [reply]
Awesome
